cloud/blockstore/libs/storage/volume/volume_actor_monitoring.cpp

```diff
@@ -1606,8 +1606,8 @@ void TVolumeActor::HandleHttpInfo_RenderNonreplPartitionInfo(
                     }
                 }
 
-                auto& infos = State->GetPartitionStatInfos();
-                if (infos && infos.front().LastCounters) {
+                auto infos = State->GetPartitionStatInfos();
```

Comment:
you might want to change other places where there is call to `State->GetPartitionStatInfos()` to use `auto infos` instead of `auto& infos`
```
./cloud/blockstore/libs/storage/volume/volume_actor_stats.cpp:148:25:    auto& info = State->GetPartitionStatInfos().front();
./cloud/blockstore/libs/storage/volume/volume_actor_stats.cpp:283:29:    for (auto& info: State->GetPartitionStatInfos())
./cloud/blockstore/libs/storage/volume/volume_actor_stats.cpp:402:29:    for (auto& info: State->GetPartitionStatInfos()) {
```

Reply:
порпавил странный паттерн State->GetPartitionStatInfos().front();
А вот остальное валидно - мы итерируем по объектам и меняем их.